### PR TITLE
Update appveyor.yml

### PR DESCRIPTION
**Pull request by NekokoTime**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -6,9 +6,9 @@ clone_depth: 3
 before_build:
   - ps: $env:Date="$(git show -s --date=short --format='%ad')"
   - appveyor DownloadFile http://releases.llvm.org/5.0.0/llvm-5.0.0.src.tar.xz
```
Comment (hcorion) [inline appveyor.yml]: You didn't change the two 5.0.0 here to 6.0

Comment (NekokoTime) [reply appveyor.yml]: Pull request #2 - Create README.md

Comment (NekokoTime) [reply appveyor.yml]: sorry my mistake when changing code

Comment (NekokoTime) [reply appveyor.yml]: esboco 18

**Pull request closed without merging**